COPYRIGHT-NOTICES.md

```diff
@@ -0,0 +1,138 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+
+# Copyrights and OpenJS Foundation projects
+
+## Ownership of Copyrights in OpenJS Foundation Contributions
+
+When source code, documentation and other content is contributed to an Open JS Foundation
+project, the copyrights in those contributions remain owned by the original
+copyright holders.
+
+The copyrights are not _assigned_ to OpenJS Foundation. Instead, they are _licensed_ for
+distribution as part of the project. Whether a project uses the DCO or a CLA,
+the original copyright holders retain their copyrights.
+
+## Copyright Notices
+
+OpenJS Foundation does not require or recommend that every contributor include their
+copyright notice in contributed files. [See below for more details on why
+not.](#why-not-list-every-copyright-holder)
+
+Instead, OpenJS Foundation recommends using a more general statement in a form similar to the
```

Comment:
Where should this copyright statement live? In `COPYRIGHT.md`? `README.md`? Another file?

Reply:
To the extent that a project wishes to add a new copyright statement, it can go in any of the above, or none of the above.  Given that adding a copyright line is optional I might suggest not getting too prescriptive about it, my concern being that people may wonder if they need to move lines around.  That's a situation we want to avoid, as it is ripe for error.

That said, LICENSE would be a great option, as many license texts include a template copyright line and it's a logical place to look.

But again, just to be sure, I'd suggest we hold off on being too explicit here, as it could be interpreted counter to "it is optional, add it if you feel you need to, but be absolutely certain not to change anything that's already there."

Reply:
I would agree with @tobie ([comment](https://github.com/openjs-foundation/cross-project-council/pull/414#issuecomment-571690635)) in that I _want_ the directions to be prescriptive, _so I don't have to think about it_--even if the details are somewhat arbitrary.